.eslintrc.js

```diff
@@ -19,12 +19,12 @@ module.exports = {
     ecmaVersion: 2018,
   },
   plugins: [
-    ‘react’,
+    'react',
   ],
   rules: {
-    ‘react/jsx-filename-extension’: [1, { extensions: [‘.js’, ‘.jsx’] }],
-    ‘import/extensions’: 0,
-    ‘react/prop-types’: ‘off’,
-    ‘react/destructuring-assignment’: [0, ‘always’],
+    'react/jsx-filename-extension': [1, { extensions: ['.js', '.jsx'] }],
+    'import/extensions': 0,
+    'react/prop-types': 'off',
+    'react/destructuring-assignment': [0, 'always'],
   },
 };
```

Comment:
eslint file doesn't work without double-quotes

Reply:
Good to know! Thanks